users.md

```diff
@@ -60,6 +60,17 @@ CarGurus is on a mission to build the world's most trusted and transparent autom
 
 Compass is a tech-driven real estate platform. With an elite team of real estate, technology and business professionals, we aim to be the best and most trusted source for home seekers.
 
+### [Cookies](https://cookies.co)
+
+<img src="https://img.cookies.co/static/logo/cookies-blue.png" width="150" align="right">
+
+COOKIES is one of the most well-respected and top-selling cannabis brands in California and throughout the world. The company and its product are recognized globally and offer a stable of over 85 cannabis varieties and product lines including indoor and sungrown flower, pre-rolls, gel caps, and vape carts. COOKIES' seed-to-sale business allows for complete quality control at every step—from cultivation and production to customers' end retail experience. Cookies has expanded to 33 stores in seven states and three countries, with 20+ more planned to open by the end of 2021.
```

Comment:
This feels a little too market-y.
And let's keep it time-insensitive, so no plans for the future, unless they are about Bazel.

Reply:
will fix :) thank you for the review